### ISSUE TITLE
Error in opening movie files

### ISSUE THREAD

**Issue by juanfadev** [atduskgreg/opencv-processing#104]
Error when trying to open a movie. Just tried with the example 'OpticalFlow' and it is imposible.

Failing in row: opencv.loadImage(video);

> Width (0) and height (0) cannot be <=0

**Comment by f41ardu** [comment]
This may solve your issue.

```
 while (video.height == 0 )  delay(10); // <-- add  
 opencv.loadImage(video);
```

or
```
 if (video.width > 0 && video.height > 0) {//check if the cam instance has loaded pixels
    opencv.loadImage(video);
  } 
```
I have no idea why this is necessary. 
Solved it for my using Processing 3.3.6.

**Comment by sean369see** [comment]
This above two modifications also work for Processing 3.3.7.
